src/containers/Tenant/Schema/SchemaViewer/SchemaViewer.tsx

```diff
@@ -82,24 +149,54 @@ export const SchemaViewer = ({keyColumnIds = [], columns = [], type}: SchemaView
                 return undefined;
             },
         },
-    ];
+    );
 
-    if (isExternalTable(type)) {
-        // External tables don't have key columns
-        dataTableColumns = dataTableColumns.filter(
-            (column) => column.name !== SchemaViewerColumns.key,
+    if (withFamilies) {
+        dataTableColumns.push(
+            {
+                name: SchemaViewerColumns.familyName,
+                width: 100,
+                sortable: false,
+                render: ({row}) => (row.Family ? families[row.Family].Name : undefined),
+            },
+            {
+                name: SchemaViewerColumns.preferredPoolKind,
+                width: 100,
+                sortable: false,
+                render: ({row}) =>
+                    row.Family
+                        ? families[row.Family].StorageConfig?.Data?.PreferredPoolKind
+                        : undefined,
+            },
+            {
+                name: SchemaViewerColumns.columnCodec,
+                width: 100,
+                sortable: false,
+                render: ({row}) => (row.Family ? families[row.Family].ColumnCodec : undefined),
+            },
```

Comment:
Added sorting. But I got a bit confused with comparators. Maybe there is a simple way.

Reply:
You don't need custom compare function in here. DataTable can compare values by `sortAccessor` (you can look how it's done for key column)

Reply:
It didn't work properly. My guess is that because of `undefined`.

Reply:
It works just fine. If you want empty fields to be sorted as well (not to stick to end of list - default behaviour), you could set empty strings as accessors instead of `undefined`. Could you show you version, maybe something wrong there?

Reply:
Examples:
```
sortAccessor: (row) => row.Family ? families[row.Family].Name : ''

sortAccessor: (row) => row.Family ? families[row.Family].StorageConfig?.Data?.PreferredPoolKind : ''

sortAccessor: (row) => row.Family ? formatColumnCodec(families[row.Family].ColumnCodec) : ''
```

Reply:
It works but in reverse order:
<img width="154" alt="image" src="https://github.com/ydb-platform/ydb-embedded-ui/assets/6786342/02b5aba3-40db-4e3a-a665-0ce5bafe5616">
I want to get order  `'a'` -> `'b'`  -> `''` and get `''` -> `'a'` -> `'b'`.

Reply:
My bad. Found a way. Fixed.